src/chain/signer.hpp

```diff
@@ -16,7 +17,10 @@ struct Signer {
 
   // TODO: Storage에서 signer가 신규인지 아닌지 검색할 수 있는 기능 추가되면
   // 제거할 것
-  bool isNew() { return true; }
+  bool isNew() {
+    auto cert = Storage::getInstance()->findCertificate(pk_cert);
```

Comment:
```suggestion
    auto cert = Storage::getInstance()->findCertificate( user_id );
```
Storage 클래스에서는 findCertifcate에서 user_id를 string으로 받습니다. (base64인코딩되어 있습니다.) Storage 클래스에 uint64_t를 받았을 때 base64 인코딩해서 처리하는 부분을 넣어야할 것으로 보입니다.

Reply:
임성균씨한테는 std::string Storage::findCertificate(uint64_t user_id)를 추가하도록 요청해두었습니다.